### PR TITLE
Style CommentForm & update PostForm

### PR DESCRIPTION
**Pull request by estherchee**
- Style comment form
- Rename css classNames for reusability
- Create ResponsiveTextArea so the placeholder auto-fit content
- Limit the input content to 255 characters
- Display the input characters below the textArea

Technical Requirements
- Style CommentForm
- Style PostForm

Screenshot:
Screenshot 2021-05-11 at 14 00 11

### CODE REVIEW

File: frontend/src/components/ResponsiveTextArea.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { useRef, useEffect, useState } from 'react';
```
Comment (eeels22) [inline frontend/src/components/ResponsiveTextArea.jsx]: Please add `// NPM Packages` above this import line

File: frontend/src/components/ResponsiveTextArea.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+export default function AutoFitContentPlaceholder({
+  placeholder,
+  contentText,
+  onFormContentChange,
+}) {
+  // Local State
+  const textArea = useRef(null);
+
+  //Method
```
Comment (eeels22) [inline frontend/src/components/ResponsiveTextArea.jsx]: Please add space between `//` and `Method`

File: frontend/src/components/ResponsiveTextArea.jsx (hunk 1 above)
Comment (eeels22) [reply frontend/src/components/ResponsiveTextArea.jsx]: `useState` can be removed from the import as it doesn't appear to be used

File: frontend/src/components/ResponsiveTextArea.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  }, [contentText]);
+
+  return (
+    <textarea
+      ref={textArea}
+      className="form-input"
+      placeholder={placeholder}
+      value={contentText}
+      onChange={(e) => onFormContentChange(e.target.value)}
+      maxlength="255"
```
Comment (eeels22) [inline frontend/src/components/ResponsiveTextArea.jsx]: I wonder if we can pass the `maxLength` as props, to make this more reusable, e.g. the user `description` has max length of 1000

**Review by eeels22 — COMMENTED** [review]
Looks great with the character limit counter! Just some minor documentation fixes and a suggestion of making the ResponsiveTextArea accept length as props rather than hardcoding 255

File: frontend/src/components/ResponsiveTextArea.jsx (hunk 3 above)
Comment (janko101) [reply frontend/src/components/ResponsiveTextArea.jsx]: I think it is a default property of textarea

**Review by janko101 — APPROVED** [review]
Nice touch with the counter

File: frontend/src/components/ResponsiveTextArea.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  }, [contentText]);
+
+  return (
+    <textarea
+      ref={textArea}
+      className="form-input"
+      placeholder={placeholder}
+      value={contentText}
+      onChange={(e) => onFormContentChange(e.target.value)}
+     />
```
Comment (eeels22) [inline frontend/src/components/ResponsiveTextArea.jsx:36]: do we need to add maxLength={maxLength}?

Comment (estherchee) [reply frontend/src/components/ResponsiveTextArea.jsx:36]: Yes

**Review by eeels22 — APPROVED** [review]
Looks great, thanks Pei!

**Pull request merged**